src/openforms/registrations/service.py

```diff
@@ -49,7 +49,6 @@ def extract_submission_reference(submission: Submission) -> str:
 
 
 def get_registration_plugin(submission: Submission) -> BasePlugin | None:
-
```

Comment:
I don't see why this change was necessary - please refrain from changing files that are not related to your ticket, as they needlessly clutter the diff for reviewers.